src/addon.h

```diff
@@ -19,8 +19,8 @@
 //#define LOG(msg) fprintf(stderr, "%s\n", msg);
 //#define TRACEF(format, arg) fprintf(stderr, format, arg);
 
-#define LOG(msg) ;
```

Comment:
These two lines shouldn’t be changed.

Reply:
reverted back to old format with a whitespace before the semicolon ^^